packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -179,6 +179,14 @@ export function makeVatKeeper(
     }
   }
 
+  function markAsDead() {
+    storage.set(`${vatID}.dead`, true);
+  }
+
+  function isDead() {
+    return storage.get(`${vatID}.dead`);
```

Comment:
.. and so maybe this should be `!!storage.get(..)` to indicate the booleanness of the return value, even though the underlying store's value is a string